lib/pdf_fill/forms/va210781v2.rb

```diff
@@ -659,10 +676,14 @@ def merge_fields(_options = {})
 
         if @form_data['events']&.any?
           process_reports
-          expand_collection('events', :format_event, 'eventOverflow')
+          expand_collection('events', :format_event, 'eventOverflow') unless options[:extras_redesign]
```

Comment:
When using ExtrasGeneratorV2, we _don't_ want to also send the multi-line concatenation to overflow.